requirements.txt

```diff
@@ -0,0 +1,22 @@
+Brotli==1.1.0
+certifi==2024.8.30
+charset-normalizer==3.3.2
+googleapis-common-protos==1.65.0
```

Comment:
I ran the pip freeze > requirements.txt command when trying to get the code up and running and so that is why it was there in the first place. I left it because I saw it in bills code also : [https://github.com/williamjhyland/viam-video-replay/blob/main/requirements.txt](https://github.com/williamjhyland/viam-video-replay/blob/main/requirements.txt) and I had trouble running the program without it

Reply:
I think it's likely okay

Generally though you only include deps you directly depend on and let their deps be taken care of in their own dep management as indirect deps

Reply:
Actually maybe we should do some investigating because it's not good to freeze their deps

In the case where Python SDK or OpenCV needed to upgrade (or downgrade) to a diff dep version, we'd be stuck at the frozen bad version

Reply:
Is there a way to do this without manually going through and creating a requirements.txt file? I used `pip freeze` the first time so I just tried` pip-chill` but it ends up giving me weird packages I didn't even have before. Any tips ?